test/specs/kefir-observable.js

```diff
@@ -0,0 +1,49 @@
+let {Kefir} = require('../test-helpers.coffee');
+
+describe('Kefir.Observable', function() {
+    describe('observe', function() {
+        let em, count, obs, sub;
```

Comment:
I can look into transpiling it, but are you running the tests in a browser or node? Because node has had support for `let` and arrow functions since [Node 4](http://node.green/).